tests/manage/monitoring/prometheusmetrics/test_rgw.py

```diff
@@ -56,3 +56,14 @@ def test_ceph_rgw_metrics_after_metrics_exporter_respin(rgw_deployments):
         "so that the list of metrics without results is empty."
     )
     assert list_of_metrics_without_results == [], msg
+
+
+@tier1
+@pytest.mark.post_ocp_upgrade
+@pytest.mark.run(order=1)
+@pytest.mark.polarion_id("OCS-2584")
+def verify_rgw_pods_restart_count(verify_rgw_restart_count_session):
```

Comment:
not sure it's a good practice to call a session scope fixture from a test function. Does it work?

Reply:
it does